docs/quickstart.rst

```diff
@@ -18,43 +18,59 @@ This quickstart guide will walk you through how to install and run NEXUS on your
 Prerequisites
 ==============
 
-* Docker (tested on v18.03.1-ce)
+* Docker (tested on v20.10.17)
 * Internet Connection
-* bash
+* bash or zsh
 * cURL
-* 500 MB of disk space
+* 10.5 GB of disk space
```

Comment:
😱 why is so much disk space needed??

Reply:
Lot of docker images that are quite sizable.

+ a little margin for granule data &c so I just rounded up to a half GB or so